chapters/object_types/volumes.txt

```diff
@@ -77,3 +77,28 @@ surface. Implementations may limit opacity to a value slightly smaller than one.
 A possible implementation of a rendering algorithm for this volume type is given
 in <<transferFunction1D_algorithm>>.
 ====
+
+[[object_types_volume_transfer_function2d]]
+==== TransferFunction2D
+
+Extension `KHR_VOLUME_TRANSFER_FUNCTION2D`
+
+A 2D transfer function uses two separate scalar fields to determine the color and opacity of the
+sampled `value`. This allows for a more nuanced and detailed visualization of volume data compared
+to 1D transfer functions. The 2D transfer function volume is created by passing the subtype string
+`transferFunction2D` to <<object_types_volume, `anariNewVolume`>>. It supports the following parameters:
+
+.<<api_concepts_parameters, Parameters>> understood by the transferFunction2D volume.
+[cols="<3,<5,>4,<7",options="header,unbreakable"]
+|===================================================================================================
+| Name             | Type                           | Default | Description
+| value            |`SPATIAL_FIELD`                 |         | <<object_types_spatial_field, Spatial field>> used for the scalar values of the volume
+| valueRange       |`FLOAT32_BOX1` / `FLOAT64_BOX1` |  [0, 1] | sampled values of `value` are clamped to this range
+| lookupTable      |ARRAY2D of <<Color>>            |         | two separate scalar fields are represented by the image width and height (see appendix for additional details)
+| unitDistance     |`FLOAT32`                       |     1.0 | distance after which a 'opacity' fraction of light traverling through the volume is absorbed
+|===================================================================================================
+
+The `lookupTable` maps the scalar `value` to color and opacity for two separate scalar
+fields. Typically, the image width (X-axis) represents the scalar `value` clamped to 
```

Comment:
I think we have to be more prescriptive here (otherwise users and implementors have a tough time): `value` *is* used for `x` and `grad(value)` *is* used for `y`.
Then, does the 2D TFN have to be separable (combination of two 1D TFNs)? Could be interesting (visually) to leave more freedom here (no difference to an implementation). Otherwise it would be better to provide the two 1D TFNs directly (and the combination to a 2D LUT is implementation choice; combining two 1D lookups would work as well and needs less memory).